### PR TITLE
Write Unit Tests of HeparinPatientHistoryEntriesControllerTest

### LINKED ISSUE #13: Write Unit Tests of HeparinPatientHistoryEntriesControllerTest
Write unit tests of HeparinPatientHistoryEntriesControllerTest.
Test all controller methods.
You can get inspiration in LoginControllerTest.

### PR DESCRIPTION
**Pull request by carishkaa**
Closes #13

### CODE REVIEW

File: backend/src/test/kotlin/blue/mild/breviary/backend/controllers/HeparinPatientHistoryEntriesControllerTest.kt [hunk 1]
```diff
@@ -1,4 +1,130 @@
...
+            "$apiPrefix/${ApiRoutes.HEPARIN_PATIENT_HISTORY_ENTRIES}/${patient.id}",
+            setOf(HttpStatus.OK),
+            typeReference<List<HeparinPatientHistoryEntryDtoOut>>(),
+            getAuthHeaders()
+        )
+
+        val historyEntriesBody = requireNotNull(historyEntries.body)
+
+        assertEquals(1, historyEntriesBody.size)
+        assertEquals(1.1f, historyEntriesBody[0].aptt)
```
Comment (marazt) [inline backend/src/test/kotlin/blue/mild/breviary/backend/controllers/HeparinPatientHistoryEntriesControllerTest.kt:82]: Ta `1.1f` je vlaste `recommendation. currentAptt` ne?

**Review by marazt — COMMENTED** [review]
Super. Jenom bych nahradil tu hodnotu na radku 82 a je to ok.

File: backend/src/test/kotlin/blue/mild/breviary/backend/controllers/HeparinPatientHistoryEntriesControllerTest.kt (hunk 1 above)
Comment (carishkaa) [reply backend/src/test/kotlin/blue/mild/breviary/backend/controllers/HeparinPatientHistoryEntriesControllerTest.kt:82]: podle me se to nevraci s tim `recommendation`, ten `currentAptt` pouze posilame s pozadavkem.
ale asi ho muzeme ziskat pres `HeparinPatientWithData` get pozadavek pres `actualAptt`, mam to tam pridat?

Comment (marazt) [reply backend/src/test/kotlin/blue/mild/breviary/backend/controllers/HeparinPatientHistoryEntriesControllerTest.kt]: Nne, to by melo byt ok.

**Review by marazt — APPROVED** [review]

**Pull request merged**